user_docs/en/userGuide.t2t

```diff
@@ -1437,7 +1437,7 @@ These actions are available under the viewer's Log menu.
 For sighted software developers or people demoing NVDA to sighted audiences, a floating window is available that allows you to view all the text that NVDA is currently speaking.
 
 To enable the speech viewer, check the "Speech Viewer" menu item under Tools in the NVDA menu.
-Uncheck the menu item to disable it.
+Uncheck the menu item to disable it. The speech viewer window contains a check box labeled "Show Speech Viewer on Startup". If this is checked, will result in the speech viewer opening when NVDA is started. The speech viewer window will always attempt to re-open with the same dimensions and location as when it was closed.

```

Comment:
1. Split the stuff about the startup check box into a new paragraph.
2. Each sentence should be on its own line to make life easier for translators. (T2t will still treat them as a continuous paragraph.)
3. The "Show Speech Viewer on Startup" check box should only have its first letter capitalised (rather than using title case) for consistency with the rest of the options in the GUI.
4. Change this sentence:

 `If this is checked, will result in the speech viewer opening when NVDA is started.`

 to:

 `If this is checked, the speech viewer will open when NVDA is started.`